server/src/l10n.js

```diff
@@ -6,98 +6,84 @@ const negotiateLanguages = require("fluent-langneg/compat");
 const { MessageContext } = require("fluent/compat");
 const mozlog = require("./logging").mozlog("l10n");
 
-let userLangs;
-exports.userLangs = userLangs;
-const contexts = [];
 let inited = false;
 
-exports.init = function(userLocales = []) {
+const rawStrings = {};
+
+exports.init = function() {
   if (inited) {
     return Promise.resolve();
   }
   inited = true;
-  exports.userLangs = userLocales;
```

Comment:
there's a possibility of a dogpile here, if multiple requests come in early. It should sort itself out, but I think it would be safer to do:

```
let initPromise;
exports.init = function() {
  if (initPromise) {
    return initPromise;
  }
  initPromise = globby(...);
  return initPromise;
};
```